src/js/lib/ajax.js

```diff
@@ -101,6 +103,8 @@ var ajax = function(opt, success, failure) {
       try {
         if (opt.type === 'json') {
           body = JSON.parse(body);
+        } else if (op.type === 'xml') {
```

Comment:
Alright, I see that it's merged. That's great, but when we try something like [this](http://jsfiddle.net/jackdalton/599u9x59/) without that `else if` where I made my typo, the plain XML text is returned, rather than the XML document. Should I create a new PR, or have you got it?